instrumentation/tapestry-5.4/javaagent/src/test/groovy/TapestryTest.groovy

```diff
@@ -54,6 +54,25 @@ class TapestryTest extends AgentInstrumentationSpecification implements HttpServ
     return "/jetty-context"
   }
 
+  WebClient client
+
+  def setup() {
+    client = WebClient.builder(address)
+      .decorator(new DecoratingHttpClientFunction() {
+        // https://github.com/line/armeria/issues/2489
```

Comment:
I decided to copy-paste this redirect logic because there are only two locations. If a third, I'd extract - I am hoping the issue gets fixed before that though. @trustin @minwoox @ikhoon

Reply:
Thanks for reminding me. 😅
Yeah, we will do. 😄